src/ConnectedDrive.ts

```diff
@@ -23,66 +24,75 @@ export class ConnectedDrive {
     async getVehicles(): Promise<Vehicle[]> {
         this.logger?.LogInformation("Getting vehicles");
         const params = `apptimezone=${120}&appDateTime=${Date.now()}`;
-        const url: string = `https://${Constants.ServerEndpoints[this.account.region]}${Constants.getVehicles}?${params}`;
-        return (await this.request(url));
+        let result: Vehicle[] = [];
+        for (let key in CarBrands) {
+            const brand: CarBrands = CarBrands[key as keyof typeof CarBrands];
+
+            this.logger?.LogInformation(`Getting ${brand} vehicles`);
+
+            const url: string = `https://${Constants.ServerEndpoints[this.account.region]}${Constants.getVehicles}?${params}`;
+            let vehicles = await this.request(url, brand);
+            result.push(...vehicles);
+        }
+        return (result);
     }
 
-    async getVehicleStatus(vin: string): Promise<VehicleStatus> {
+    async getVehicleStatus(vin: string, brand: CarBrands): Promise<VehicleStatus> {
```

Comment:
Add these as optional params with BMW as the default value, otherwise the API will be broken. Same for the rest of the methods as well.

Reply:
Good idea, I doubted to do that initially, but made it a required parameter since it would be a new version anyway. I've made the changes so it is optional and it uses BMW as default.